backend/alembic/versions/871e5285a2a3_add_name_for_glossary_doc.py

```diff
@@ -0,0 +1,40 @@
+"""add_name_for_glossary_doc
+
+Revision ID: 871e5285a2a3
+Revises: bc813d99ccfb
+Create Date: 2024-08-15 22:11:42.150076
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pylint: disable=E1101
+
+# revision identifiers, used by Alembic.
+revision: str = "871e5285a2a3"
+down_revision: Union[str, None] = "bc813d99ccfb"
```

Comment:
```suggestion
down_revision: Union[str, None] = "96bf2e203c03"
```
As my PR was merged.

Reply:
Fixed